src/GitVersion.Core/Configuration/BranchConfigurationCalculator.cs

```diff
@@ -132,7 +132,7 @@ private BranchConfig InheritBranchConfiguration(IBranch targetBranch, BranchConf
                 {
                     // TODO We should call the build server to generate this exception, each build server works differently
                     // for fetch issues and we could give better warnings.
-                    throw new InvalidOperationException("Could not find a 'develop' or 'main' branch, neither locally nor remotely.");
+                    throw new InvalidOperationException("Gitversion could not determine which branch to treat as the development branch (default = develop) nor release branch (default = main or master), neither locally nor remotely. Ensure the local clone and checkout match the requirements or considering using 'Gitversion Dynamic Repositories'");
```

Comment:
```suggestion
                    throw new InvalidOperationException("Gitversion could not determine which branch to treat as the development branch (default 'develop') nor releasable branch (default 'main' or 'master'), neither locally nor remotely. Ensure the local clone and checkout match the requirements or considering using 'GitVersion Dynamic Repositories'");
```

Reply:
I also didn't like the "neither" "nor" structure so I did a commit that incorporated your changes exactly, but also resolved the neither nor - which was a double negative I think.